ros2pkg/ros2pkg/resource/cmake/CMakeLists.txt.em

```diff
@@ -0,0 +1,99 @@
+cmake_minimum_required(VERSION 3.5)
+project(@(project_name))
+
+# Default to C99
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 99)
+endif()
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+# find dependencies
+@[if dependencies]@
+@[for deb in dependencies]@
+find_package(@deb REQUIRED)
+@[end for]@
+
+include_directories(
+@[if create_cpp_library]@
+  include
+@[end if]@
+@[for deb in dependencies]@
+  ${@(deb)_INCLUDE_DIRS}
+@[end for]@
+)
+
+@[else]@
+# uncomment the following section in order to fill in
+# further dependencies manually.
+# find_package(<dependency> [<REQUIRED>|<QUIET>])
+
+# include_directories($<dependency>_INCLUDE_DIRS})
+
+@[end if]@
+@[if create_cpp_library]@
+add_library(${PROJECT_NAME} src/@(cpp_library_name))
```

Comment:
in the case of pure cmake packages they need to be built as shared libraries or to add the "BUILD_SHARED" option